node_exporter.go

```diff
@@ -137,7 +140,19 @@ func (h *handler) innerHandler(filters ...string) (http.Handler, error) {
 }
 
 func main() {
+
+	done := make(chan os.Signal, 1)
+	signal.Notify(done, syscall.SIGINT, syscall.SIGTERM)
+
 	var (
+		socketPath = kingpin.Flag(
+			"web.socket-path",
+			"Path to unix socket file.",
```

Comment:
not important:

    Path to a unix socket file on which to export metrics and web interface

(to be consistent with web.listen-address description)